### DISCUSSION TITLE
Installation 2.4.20 failes with errors

### DISCUSSION THREAD

**Discussion by ZRHSecOnion** [Security-Onion-Solutions/securityonion#11629] [2.4]
### Version

2.4.20

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

4

### RAM

16GB

### Storage for /

100GB

### Storage for /nsm

100GB

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Hello - I try to install a ManagerNode on 2.4.20. The installation failles all the time. 
I get this error: 
      Result: False
                  

> Failed to query for current Logstash Outputs...

      Result: False
                  

> Failed to query for current Fleet Server Elasticsearch URLs...

any guess?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by ZRHSecOnion** [comment]
on our Proxy I se many of this: 

**http://10.xy.ab.tz:7000/playbook/webhook**

the no-proxy parameter seems to be missing

**Comment by ZRHSecOnion** [comment]
on our Proxy I found a Denied to

https://securitycheck.phusionpassenger.com

has this to be addad?

**Comment by dougburks** [comment]
If you're having proxy problems, you can try the airgap option to see if that works better:
https://docs.securityonion.net/en/2.4/airgap.html
